plugin/advisor/mysql/advisor_stmt_no_leading_wildcard_like.go

```diff
@@ -0,0 +1,99 @@
+package mysql
+
+import (
+	"fmt"
+
+	"github.com/bytebase/bytebase/common"
+	"github.com/bytebase/bytebase/plugin/advisor"
+	"github.com/bytebase/bytebase/plugin/db"
+	"github.com/pingcap/tidb/parser/ast"
+	"github.com/pingcap/tidb/parser/format"
+)
+
+const (
+	wildcard byte = '%'
```

Comment:
Use string instead of byte for readability since we are doing string-to-string comparison. Also we don't need to worry about performance.